sdk/src/main/java/io/customer/sdk/data/model/CustomAttributes.kt

```diff
@@ -0,0 +1,30 @@
+package io.customer.sdk.data.model
```

Comment:
We can probably move this to the `base` module so, the `messaging` module can use it as well?

Reply:
I spent some time doing this but ran into issues since I wrote tests for `CustomAttributes` but `base` does not have tests setup for it. Since it's a java module and not Android, I couldn't just copy over the running of android tests configuration from other modules. 

I agree this could be moved. When we [write javadoc documentation](https://github.com/customerio/issues/issues/6676) for the SDK, I believe that this file could be moved then. So that `public` facing code that isn't meant to be exposed to customers (like this file) could be moved to another module and therefore, the generated javadoc only shows `public` code that we expect to show customers. That's one idea of when we could move this file in future work. 

Do you think we should do the work then? Or, do you have an idea for how to setup java-only automated tests now and get this moved now?